modules/xmpp/SdpTransformUtil.js

```diff
@@ -0,0 +1,439 @@
+import * as transform from 'sdp-transform';
+
+/**
+ * Parses the primary SSRC of given SSRC group.
+ * @param {object} group the SSRC group object as defined by the 'sdp-transform'
+ * @return {Number} the primary SSRC number
+ */
+export function parsePrimarySSRC(group) {
+    return parseInt(group.ssrcs.split(" ")[0]);
+}
+
+/**
+ * Parses the secondary SSRC of given SSRC group.
+ * @param {object} group the SSRC group object as defined by the 'sdp-transform'
+ * @return {Number} the secondary SSRC number
+ */
+export function parseSecondarySSRC(group) {
+    return parseInt(group.ssrcs.split(" ")[1]);
+}
+
+function findGroup(mLine, groupSemantics, ssrcs) {
+    return mLine.ssrcGroups && mLine.ssrcGroups.find(
+            (group) => group.semantics === groupSemantics
+                            && !ssrcs || ssrcs === group.ssrcs);
+}
+
+function findGroupByPrimarySSRC(mLine, groupSemantics, primarySsrc) {
+    return mLine.ssrcGroups && mLine.ssrcGroups.find(
+            (group) => group.semantics === groupSemantics
+                            && parsePrimarySSRC(group) === primarySsrc);
+}
+
+function getSSRCCount(mLine) {
+    if (!mLine.ssrcs) {
+        return 0;
+    } else {
+        return mLine.ssrcs
+            .map(ssrcInfo => ssrcInfo.id)
+            .filter((ssrc, index, array) => array.indexOf(ssrc) === index)
+            .length;
+    }
+}
+
+/**
+ * Utility class for SDP manipulation using the 'sdp-transform' library.
+ *
+ * Typical use usage scenario:
+ *
+ * const transformer = new SdpTransformWrap(rawSdp);
+ * if (transformer.selectMedia('video)) {
+ *     transformer.addSSRCAttribute({
+ *         id: 2342343,
+ *         attribute: "cname",
+ *         value: "someCname"
+ *     });
+ *     rawSdp = transformer.toRawSdp();
+ * }
+ */
+export class SdpTransformWrap {
```

Comment:
Ok, so I've pushed another commit which extracts separate instance called MLineWrap to get rid of this weird "select m line" state. I think it's better compared to static methods for which I've opened a separate PR#382, because:
1. There's no real benefit of functional approach/stateless  unless you're doing parallel processing
2. You have to pass the m line instance to almost every single utility call, so in my opinion it's more convenient to have a wrapper object around that m line.